Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -373,15 +373,21 @@ div.bocList div.bocListTableBlock div.bocListNavigator a:not([href]) img
 /*endregion*/
 
 /*region validation marker*/
-div.bocList .bocListCellStructureElement > .validationErrorMarker
-{
-  margin-top: calc(var(--remotion-themed-spacing) / 2);
-}
-
 div.bocList .bocListCellStructureElement > .validationErrorMarker,
 div.bocList .bocListContent > .validationErrorMarker
 {
+  padding: calc(var(--remotion-themed-spacing) / 2);
```

Comment:
```suggestion
  padding: calc(var(--remotion-themed-spacing-small) - var(--remotion-themed-border-width));
```
Also, both versions result in the ValidationColumn Icon to go below the baseline. The selector might be too generic or there is another sideeffect.

![image](https://github.com/re-motion/Framework/assets/6972548/d0b5d51c-4211-424a-9584-fdb2fc33105f)

Reply:
How did I miss that 😅 The first marker was in a `.bocListContent` which already applies a padding so it was added twice.